src/screens/home.js

```diff
@@ -0,0 +1,6 @@
+const Home = props => (
```

Comment:
Nitpicking but capitalise the filename to be consistent.

Reply:
In the course I am currently doing, he make presentational components first letter uppercase then screens as lowercase... that's why I've done this.

Reply:
Cool, I'm ok with that as long as we're consistent across this codebase.